source/user-manual/reference/ossec-conf/active-response.rst

```diff
@@ -161,7 +161,7 @@ Sets timeouts in minutes for repeat offenders. This is a comma-separated list of
 +--------------------+-----------------------------+
 
 .. warning::
-    This option must be configured directly in the **ossec.conf** file of the agent, even when using a manager/agent setup with centralized configuration of other settings via **agent.conf**. Apart from that, it has to be defined in the upper ``<active-response>`` section found in the configuration file.
+    This option must be configured directly in the **agent.conf** file of the agent, even when using a manager/agent setup with centralized configuration of other settings via **agent.conf**. Apart from that, it has to be defined in the upper ``<active-response>`` section found in the configuration file.
```

Comment:
```suggestion
    This option must be configured directly in the **agent.conf** file of the agent, even when using a manager/agent setup with centralized configuration of other settings via **shared/agent.conf**. Apart from that, it has to be defined in the upper ``<active-response>`` section found in the configuration file.
```

Reply:
could be something like?

_This option must be configured in the **agent.conf** file in the agent, and also in the **manager.conf** file in manager side, even when using a manager/agent setup with centralized configuration of other settings via **shared/agent.conf**. Apart from that, it has to be defined in the upper ``<active-response>`` section found in the configuration file._